google/auth/transport/aiohttp_requests.py

```diff
@@ -264,7 +322,7 @@ async def request(
                     data=data,
                     headers=request_headers,
                     timeout=timeout,
-                    **kwargs
+                    **kwargs,
```

Comment:
Get rid of stray comma here and below. In general, it's good to review your diff to make sure you're only including changes that you really want to include.

Reply:
I think this may be the autoformatter at work.  Might be worth running `nox -s blacken` across code quick actually, might be a differing version?

Reply:
confirmed this is autoformatter (black)